README.md

```diff
@@ -428,6 +428,40 @@ memory, use:
 * `export NCCL_P2P_DISABLE=1` for NCCL.
 * `--mca btl_smcuda_use_cuda_ipc 0` flag for OpenMPI and similar flags for other vendors.
 
+## Analyzing Horovod Performance
+
+Horovod has an ability to record the timeline of its activity, called Horovod Timeline.
+
+![Horovod Timeline](https://user-images.githubusercontent.com/16640218/29540325-a80e2140-8682-11e7-9f21-ada1613948be.png)
+
+In order to record Horovod Timeline, please set `HOROVOD_TIMELINE` environment variable to location of the timeline
```

Comment:
In order to record Horovod Timeline, please set `HOROVOD_TIMELINE` environment variable to location  --> To record a Horovod Timeline, set the `HOROVOD_TIMELINE` environment variable to the location